src/components/ViewDao/DaoSidebar.tsx

```diff
@@ -29,50 +32,54 @@ class DaoSidebarComponent extends React.Component<IProps, null> {
           <div className={css.daoDescription}>
-            { dao.name === "Meme" ?
-                <p><a href="https://docs.google.com/document/d/1iJZfjmOK1eZHq-flmVF_44dZWNsN-Z2KAeLqW3pLQo8" target="_blank">Learn how to MemeDAO</a></p>
+            {dao.name === "Meme" ?
+              <p><a href="https://docs.google.com/document/d/1iJZfjmOK1eZHq-flmVF_44dZWNsN-Z2KAeLqW3pLQo8" target="_blank">Learn how to MemeDAO</a></p>
               : dao.name === "Identity" ?
                 <p>
                   A curated registry of identities on the Ethereum blockchain.&nbsp;
                   <a href="https://docs.google.com/document/d/1_aS41bvA6D83aTPv6QNehR3PfIRHJKkELnU76Sds5Xk" target="_blank">How to register.</a>
                 </p>
-              : <p>Anyone can make a proposal to the DAO! Click the button on the top right.</p>
+                : <p>Anyone can make a proposal to the DAO! Click the button on the top right.</p>
             }
           </div>
           <div className={css.navigation}>
             <span className={css.navHeading}><b>Menu</b></span>
             <ul>
               <li>
                 <Link to={"/dao/" + dao.address}>
+                  <span className={css.menu_dot} />
                   <span className={css.notification + " " + css.homeNotification}></span>
-                  <img src="/assets/images/Icon/menu/home.svg"/>
+                  <img src="/assets/images/Icon/menu/home.svg" />
                   Home
                 </Link>
               </li>
               <li>
                 <Link to={"/dao/" + dao.address + "/members/"}>
-                <span className={css.notification + " " + css.holdersNotification}></span>
-                <img src="/assets/images/Icon/menu/holders.svg"/>
+                  <span className={css.menu_dot} />
+                  <span className={css.notification + " " + css.holdersNotification}></span>
+                  <img src="/assets/images/Icon/menu/holders.svg" />
                   Reputation Holders
                  </Link>
               </li>
               <li>
-                  <NavLink activeClassName={css.selected} to={"/dao/" + dao.address + "/history/"}>
-                    <span className={css.notification + " " + css.historyNotification}></span>
-                    <img src="/assets/images/Icon/menu/history.svg"/>
-                    History
+                <NavLink activeClassName={css.selected} to={"/dao/" + dao.address + "/history/"}>
+                  <span className={css.menu_dot} />
+                  <span className={css.notification + " " + css.historyNotification}></span>
+                  <img src="/assets/images/Icon/menu/history.svg" />
+                  History
                   </NavLink>
               </li>
               <li>
                 <NavLink activeClassName={css.selected} to={"/dao/" + dao.address + "/redemptions/"}>
+                  <span className={css.menu_dot + (proposalCount ? " " + css.red : "")} />
```

Comment:
lets use the classNames library to make this cleaner, we use it all across the app to handle logic for classes on elements

Reply:
I'm not sure what you're asking for here, since I am using className

Reply:
search the code for classNames (it's plural), its a library that makes it much easier to use logic to determine which classes should be on an element. we use it in almost every component

Reply:
Gotcha.  It seems that in that module there is a bunch of code unrelated to mine which is also not using classNames.  I went ahead and refactored it all to use classNames.